test/encapsulate_op/encapsulate_op_test.cc

```diff
@@ -1,131 +0,0 @@
-/*******************************************************************************
- * Copyright 2019-2020 Intel Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *******************************************************************************/
-
```

Comment:
Why remove tests?

Reply:
Primarily because of the refactoring - those interfaces that the unit test was testing for are now gone. For the most part, `GetNgExecutable` is the only helper that the kernel implementation now uses.

Unit tests for the kernel make sense but I removed the header for the kernel to adhere to the TF convention. I could add a test in the kernel directory like TF but I'm not sure if our testing is set up to handle unit tests at the same level as the source?

Reply:
I've added a task to capture this and will add a test for "Compute()" once we add inline unit tests to test the internal interfaces.